src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -564,4 +565,51 @@ public RankerMetrics Evaluate(IDataView data, string label, string groupId, stri
             return eval.Evaluate(data, label, groupId, score);
         }
     }
+
+    /// <summary>
+    /// The central catalog for anomaly detection tasks and trainers.
+    /// </summary>
+    public sealed class AnomalyDetectionCatalog : TrainCatalogBase
+    {
+        /// <summary>
+        /// The list of trainers for anomaly detection.
+        /// </summary>
+        public AnomalyDetectionTrainers Trainers { get; }
+
+        internal AnomalyDetectionCatalog(IHostEnvironment env)
+            : base(env, nameof(AnomalyDetectionCatalog))
+        {
+            Trainers = new AnomalyDetectionTrainers(this);
+        }
+
+        public sealed class AnomalyDetectionTrainers : CatalogInstantiatorBase
+        {
+            internal AnomalyDetectionTrainers(AnomalyDetectionCatalog catalog)
+                : base(catalog)
+            {
+            }
+        }
+
+        /// <summary>
+        /// Evaluates scored anomaly detection data.
+        /// </summary>
+        /// <param name="data">The scored data.</param>
+        /// <param name="label">The name of the label column in <paramref name="data"/>.</param>
+        /// <param name="score">The name of the score column in <paramref name="data"/>.</param>
+        /// <param name="predictedLabel">The name of the predicted label column in <paramref name="data"/>.</param>
+        /// <returns>The evaluation results for these calibrated outputs.</returns>
```

Comment:
>calibrated [](start = 54, length = 10)

What is calibrated here? Could you explain in a few more words?

Reply:
it was a copy-paste typo. fixed... 

---
In reply to: [255717912](https://github.com/dotnet/machinelearning/pull/2390#discussion_r255717912) [](ancestors = 255717912)